draft-ietf-rats-architecture.md

```diff
@@ -889,11 +889,9 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.
 In most cases, components that have to be vouched for via Endorsements because no Evidence is generated about them are referred to as roots of trust.
 
-The manufacturer of the Attester arranges for its Attesting Environment to be provisioned with key material.
-The key material is typically in the form of an asymmetric key pair (e.g., an RSA or ECDSA private key
-and a manufacturer-signed IDevID certificate) secured in the Attester.
+The manufacturer having arranged for an Attesting Environment to be provisioned with key material with which to sign Evidence, the Verifier is then provided with
+some way of verifying the signature on the Evidence.  This may in the form of an appropriate trust anchor, or the Verifier may be provided with a database of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
```

Comment:
```suggestion
some way of verifying the signature on the Evidence.  This may be in the form of an appropriate trust anchor, or the Verifier may be provided with a database of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
```